readme.md

```diff
@@ -58,28 +57,31 @@ This is the official DOPE ROS package for detection and 6-DoF pose estimation of
       $ roscore
       ```
 
-2. **Start camera node** (or start your own camera node)
-      ```
-      $ rosrun dope camera.py  # Publishes RGB images to `/dope/webcam_rgb_raw`
-      ```
-  
+2. **Start camera node**
+
+    Start the ROS driver for your camera. If you are using a webcam, try one of these:
 
+    * [usb_cam](http://wiki.ros.org/usb_cam)
+    * [libuvc_camera](http://wiki.ros.org/libuvc_camera)
+
+    The camera must publish a correct `camera_info` topic to enable DOPE to compute the correct poses. Basically all ROS drivers have a `camera_info_url` parameter where you can set the calibration info (but most ROS drivers include a reasonable default).
+  
 3. **Edit config info** (if desired) in `~/catkin_ws/src/dope/config/config_pose.yaml`
     * `topic_camera`: RGB topic to listen to
+    * `topic_camera_info`: camera info topic to listen to
     * `topic_publishing`: topic name for publishing
+    * `input_is_rectified`: Whether the input images are rectified. It is strongly suggested to use a rectified input topic.
     * `weights`: dictionary of object names and there weights path name, **comment out any line to disable detection/estimation of that object**
     * `dimension`: dictionary of dimensions for the objects  (key values must match the `weights` names)
     * `draw_colors`: dictionary of object colors  (key values must match the `weights` names)
-    * `camera_settings`: dictionary for the camera intrinsics; edit these values to match your camera
-    * `thresh_points`: Thresholding the confidence for object detection; increase this value if you see too many false positives, reduce it if  objects are not detected. 
-    
+    * `thresh_points`: Thresholding the confidence for object detection; increase this value if you see too many false positives, reduce it if  objects are not detected.
+    * `downscale_height`: If the input image is larger than this, scale it down to this pixel height. Very large input images eat up all the GPU memory and slow down inference. Also, DOPE works best when the object size (in pixels) has appeared in the training data (which is downscaled to 400 px). For these reasons, downscaling large input images to something reasonable (e.g., 400-500 px) improves memory consumption, inference speed *and* recognition results.
+
 4. **Start DOPE node**
     ```
-    $ rosrun dope dope.py [my_config.yaml]  # Config file is optional; default is `config_pose.yaml`
+    $ roslaunch dope dope.launch [config:=/path/to/my_config.yaml]  # Config file is optional; default is `config_pose.yaml`
     ```
 
-    *Note:*  Config files must be located in the `~/catkin_ws/src/dope/config/` folder.
```

Comment:
Because the config files don't have to be located in the `dope/config` folder any more. You can now specify an arbitrary config file by running:

```bash
roslaunch dope dope.launch config:=/path/to/my/config.yaml
```

or equivalently, if you want to specify the path relative to your own package:

```bash
roslaunch dope dope.launch config:=$(rospack find my_dope_config_pkg)/config/my_dope_config.yaml
```

Reply:
That's also what it says in the line above:

```
$ roslaunch dope dope.launch [config:=/path/to/my_config.yaml]  # Config file is optional; default is `config_pose.yaml`
```